### ISSUE TITLE
lock.getUserInfo is not a function

### ISSUE THREAD

**Issue by lezyeoh** [auth0/angular-lock#24]
As title.

No function exists and as getProfile is soon to be deprecated, I was hoping to transfer over to the new function.

**Comment by chenkie** [comment]
Can you confirm which version of Lock you're using?

**Comment by lezyeoh** [comment]
<script src="//cdn.auth0.com/js/lock/10.2.2/lock.min.js"></script>

**Comment by chenkie** [comment]
Can you try with the latest version of Lock? 10.11.0

**Comment by lezyeoh** [comment]
That's solved.  I'm getting an Auth0 is not a constructor error, but I'll investigate a bit further before coming back with anything.

**Comment by lezyeoh** [comment]
I've tried investigating this error, but found I'm still getting the Auth0 is not a constructor error.
so using version 10.2.2

on successful authentication, I've pinned the error down to the code below.

It occurs here:
```
console.log('here');
lock.on("authenticated", function(authResult) {
   console.log('here2');
}
```

Console log shows:
here
TypeError: Auth0 is not a constructor
here2

It doesn't actually seem to cause any "functional" issues as everything seems fine.  I just wanted to make sure you're aware of it in case I'm missing something important.  Plus I hate seeing errors in the console log ;)
